PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalClientUnitTest.java

```diff
@@ -506,6 +510,43 @@ public void tokenize_whenPayPalInternalClientTokenizeResult_callsBackResult()
         verify(braintreeClient).sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_SUCCEEDED, params);
     }
 
+    @Test
+    public void createEditAuthRequest_whenEditRequest_sendsPayPalEditRequestViaInternalClient() {
```

Comment:
It looks like we just have tests around the edit auth request. Do we also need tests around the public edit method similar to the tokenize tests in this file?